websockets/basic-auth.any.js

```diff
@@ -1,11 +1,11 @@
 // META: global=window,worker
-// META: script=websocket.sub.js
+// META: script=constants.sub.js
+// META: variant=?wss
+//
+// XXX: this test is not relevant for HTTP/2
```

Comment:
Why is this not relevant to HTTP/2?

Reply:
Actually, they are relevant. I needed to modify the handler, but now this test is working as expected.